### ISSUE TITLE
Gallery inside magnifier moves blocky

### ISSUE THREAD

**Issue by GrimLink** [magento/magento2#29076]
### Preconditions (*)

The magnifier moves only when the cursor is leaving the magnifier area.
Causing a blocky movement.
This behavior only happens to the magnifier when the mode option is set to inside.

_Tested on a clean Magento 2.3.5 version._

### Steps to reproduce (*)

In the view.xml for any theme.
Enable the magnifier and set the mode to inside.

```xml
<var name="magnifier">
  <var name="enabled">true</var> <!-- Turn on/off magnifier (true/false) -->
  <var name="mode">inside</var> <!-- Zoom type (outside/inside) -->
</var>
```

### Expected result (*)

A smooth movement.
Where the magnifier moves with the mouse cursor.

![Screen Recording 2020-07-10 at 22 15 23](https://user-images.githubusercontent.com/4387541/87200371-ab514a80-c2fc-11ea-8287-e7f8cbd37438.gif)

### Actual result (*)

Blocky movement as seen on this image.

![Screen Recording 2020-07-10 at 22 18 18](https://user-images.githubusercontent.com/4387541/87200348-9f658880-c2fc-11ea-85a6-916d4708b5eb.gif)

---
Please provide [Severity](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#backlog) assessment for the Issue as Reporter. This information will help during Confirmation and Issue triage processes.

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [x] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by magento-engcom-team** [comment]
@engcom-Alfa Thank you for verifying the issue.

Unfortunately, not enough information was provided to acknowledge ticket. Please consider adding the following:
- [ ] Add `"Component: "` label(s) to this ticket based on verification result. If uncertain, you may follow the best guess

Once all required information is added, please add label `"Issue: Confirmed"` again.
Thanks!

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Alfa 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-35918` were created

**Issue Available**: @engcom-Alfa, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by magento-engcom-team** [comment]
Hi @GrimLink. Thank you for your report. 
The issue has been fixed in magento/magento2#29077 by @GrimLink in 2.4-develop branch
Related commit(s):
 - [feb01e40c1d28f38412145dd70e547ffdfaa951c](https://github.com/magento/magento2/commit/feb01e40c1d28f38412145dd70e547ffdfaa951c)

The fix will be available with the upcoming 2.4.1 release.
